src/main/java/world/bentobox/bentobox/database/objects/Island.java

```diff
@@ -660,21 +683,22 @@ public boolean showInfo(User user) {
             // Fixes #getLastPlayed() returning 0 when it is the owner's first connection.
             long lastPlayed = (plugin.getServer().getOfflinePlayer(owner).getLastPlayed() != 0) ?
                     plugin.getServer().getOfflinePlayer(owner).getLastPlayed() : plugin.getServer().getOfflinePlayer(owner).getFirstPlayed();
-            user.sendMessage("commands.admin.info.last-login","[date]", new Date(lastPlayed).toString());
-
-            user.sendMessage("commands.admin.info.deaths", "[number]", String.valueOf(plugin.getPlayers().getDeaths(world, owner)));
-            String resets = String.valueOf(plugin.getPlayers().getResets(world, owner));
-            String total = plugin.getIWM().getResetLimit(world) < 0 ? "Unlimited" : String.valueOf(plugin.getIWM().getResetLimit(world));
-            user.sendMessage("commands.admin.info.resets-left", "[number]", resets, "[total]", total);
-            // Show team members
-            showMembers(user);
+                    user.sendMessage("commands.admin.info.last-login","[date]", new Date(lastPlayed).toString());
+
+                    user.sendMessage("commands.admin.info.deaths", "[number]", String.valueOf(plugin.getPlayers().getDeaths(world, owner)));
+                    String resets = String.valueOf(plugin.getPlayers().getResets(world, owner));
+                    String total = plugin.getIWM().getResetLimit(world) < 0 ? "Unlimited" : String.valueOf(plugin.getIWM().getResetLimit(world));
+                    user.sendMessage("commands.admin.info.resets-left", "[number]", resets, "[total]", total);
+                    // Show team members
+                    showMembers(user);
         }
         Vector location = center.toVector();
         user.sendMessage("commands.admin.info.island-location", "[xyz]", Util.xyz(location));
         Vector from = center.toVector().subtract(new Vector(range, 0, range)).setY(0);
         Vector to = center.toVector().add(new Vector(range-1, 0, range-1)).setY(center.getWorld().getMaxHeight());
         user.sendMessage("commands.admin.info.island-coords", "[xz1]", Util.xyz(from), "[xz2]", Util.xyz(to));
         user.sendMessage("commands.admin.info.protection-range", "[range]", String.valueOf(protectionRange));
+        user.sendMessage("commands.admin.info.max-protection-range", "[range]", String.valueOf(maxEverProtectionRange));
```

Comment:
This might only lead to admins being confused. Why aren't you using the range (island distance) instead?

Reply:
Quick answer - using island distance takes too long and is not necessary because only the protection range chunks should need to be deleted.

To avoid admin confusion, I could delete that line 701 and just keep it an internal number. 

By the way, I did some experiments with deleting large island ranges running in the background they can take a very long time to delete (minutes), so that's why I wanted to avoid having to regenerate all those chunks. Once 4 or 5 islands were in deletion, the server became very sluggish. So, it's a tradeoff.

Reply:
We can make this small change on the main develop branch if required. Let me know what you think.

Reply:
Okay, I understand why you didn't use range then. That makes sense.

Then, I'd like to remove line 701. Just to avoid any kind of "support request" that could occur because of that.